appveyor-ubuntu.yml

```diff
@@ -206,10 +200,6 @@ for:
         - configuration: LocalZipTerraformBuildInContainerIntegTesting
 
     test_script:
-      # Pre-dev Tests
-      - "pip install -e \".[pre-dev]\""
```

Comment:
I think we need this or we need to move `ruff` into the `requirements/dev.txt`. I don't know why we have a "pre-dev" file.

Reply:
As far as I understand that `pre-dev.txt` is imported at the top of the `dev.txt` https://github.com/aws/aws-sam-cli/blob/develop/requirements/dev.txt#L1

Reply:
Well would you look at that... lol